<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {

			background-color :  rgb(255, 255, 255) ; 
		
			border-radius : 5%; 		
		padding : 0 ; 
	 margin : 0 ; 


}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">
  QTabWidget::pane {
	margin : 5px  ;

			border-radius : 3% ;


        }



QTabWidget::tab-bar {
	alignment : right     ;


}

QTabBar
{

		font: 11pt &quot;Ravie&quot;;
	font-weight : bold ;

}
QTabBar::tab {

	margin-top : 10px;
	background-color : #36304A; 
	color :  rgb(157, 157, 157)   ;
	padding : 5px ;

	border : 1px solid transparent    ;
	border-radius : 3%;


}

QTabBar::tab::selected {
	background-color  :  rgb(95, 85, 131) ; 
	color : white    ; 



}



   QTabBar::tab:hover {


        }</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>تكاليف الشركات </string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>تكاليف أخري</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>طلبات المتاجر </string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_1">
      <attribute name="icon">
       <iconset resource="../../resources/resources.qrc">
        <normaloff>:/icons/icons/home.png</normaloff>:/icons/icons/home.png</iconset>
      </attribute>
      <attribute name="title">
       <string>الصفحه الرئيسية</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
